client/src/views/Game.vue

```diff
@@ -24,7 +30,17 @@
               class="ma-3 pa-6"
               outlined
               tile
-            >{{ player.name }}</v-card>
+            >
+              <div>
+                <span>Name: </span>{{ player.name }}
+              </div>
+              <div v-if="gameState.status === 'Playing'">
+                <span>Cards in Hand:</span>
+                <ul class="hand">
+                  <li v-for="(card, index) of player.cards" :key="index">🃏</li>
```

Comment:
So I really like the idea of visually showing the cards. But I wonder if this would make sense to utilize flexbox to prevent wrapping. Or just showing a number of cards instead of displaying them.